x/feemarket/keeper/keeper.go

```diff
@@ -54,12 +56,16 @@ func NewKeeper(
 		panic(err)
 	}
 
+	maxValue := new(big.Int).Sub(new(big.Int).Exp(big.NewInt(2), big.NewInt(256), nil), big.NewInt(1))
```

Comment:
might be better to use sdkmath.MaxBitLen for `2^256-1`?

Reply:
updated!